.github/workflows/publish-js-recon.yml

```
@@ -141,10 +141,17 @@ jobs:
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          echo "chore: merge changes after release $VERSION"
+          git fetch origin main:main
+          git fetch origin dev:dev
           git checkout dev
-          git merge main
+          git merge --no-ff -m "chore: merge changes after release $VERSION" main -X theirs || (git status && git diff && exit 0)
           git push origin dev
-        
+        env:
+          VERSION: ${{ github.event.release.tag_name }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)